src/components/CollectionFilterRow.vue

```diff
@@ -0,0 +1,261 @@
+<template>
+  <div class="collection-filter-row">
+    <span class="or">OR</span>
+    <FormWidget v-if="newFilter.isAdvanced" style="flex: 1">
+      <input v-model="newFilter.content" />
+    </FormWidget>
+    <template v-else>
+      <FormWidget :before="currentFilterIcon">
+        <select v-model="newFilter.builder.property">
+          <option v-if="!newFilter.builder.property" value="">
+            - Property -
+          </option>
+          <option
+            v-for="(filter, property) in availableFilters"
+            :key="property"
+            :value="property"
+          >
+            {{ filter.label ?? property }}
+          </option>
+        </select>
+      </FormWidget>
+      <template v-if="hasComparisonSelect">
+        <FormWidget v-if="currentFilter?.type === 'string'">
+          <select v-model="newFilter.builder.negate">
+            <option :value="false">does</option>
+            <option :value="true">does not</option>
+          </select>
+        </FormWidget>
+        <FormWidget v-if="hasComparisonSelect">
+          <select v-model="newFilter.builder.comparison">
+            <option
+              v-for="(label, type) in comparisons"
+              :key="type"
+              :value="type"
+            >
+              {{ label }}
+            </option>
+          </select>
+        </FormWidget>
+      </template>
+      <FormWidget
+        v-if="hasValueInput"
+        :after="valueInputAfter"
+        :before="valueInputBefore"
+      >
+        <input v-model="newFilter.builder.value" />
+      </FormWidget>
+      <template v-else-if="currentFilter?.type === 'enum'">
+        <FormWidget>
+          <select v-model="newFilter.builder.negate">
+            <option :value="false">is</option>
+            <option :value="true">is not</option>
+          </select>
+        </FormWidget>
+        <FormWidget>
+          <select v-model="newFilter.builder.value">
+            <option v-if="!newFilter.builder.value" value="" />
+            <option v-for="choice in enumChoices" :key="choice" :value="choice">
+              {{ choice }}
+            </option>
+          </select>
+        </FormWidget>
+      </template>
+    </template>
+    <UiButton
```

Comment:
should we disable button is the filter is not in a correct state , like "power_state is " and now value